cmd/cloudx/client/handler.go

```diff
@@ -153,6 +153,19 @@ func NewCommandHelper(cmd *cobra.Command) (*CommandHelper, error) {
 	}, nil
 }
 
+func (h *CommandHelper) GetDefaultProjectID() (string, error) {
```

Comment:
Apart from some tests, this error is never actually used. It is in the most cases ignored. If it's acceptable for this method to just return an empty string when no project was found, then you can remove the error all together and simplify the code a bit. If you do need the error, it would be better to define the error

```
var DefaultProjectNotSetError = errors.New("...")
```

later, you can test for the error:

```
if id, err := GetDefaultProjectID(); errors.Is(DefaultProjectNotSetError) {
  // do something
} else if err != nil {
  Fatal(err)
}
```